examples/busybox/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: busybox
+version: 1.0
+summary: busybox example to showcase the snapcraft kbuild plugin
+description:
+   this is a snapcraft example package that contains the output of the
+   snapcraft busybox example.
+
+apps:
+  ls:
+    command: ls
+    plugs: [old-security]
+  touch:
+    command: touch
+    plugs: [old-security]
+
+plugs:
+  old-security:
+    security-override:
+      read-paths:
+        - /
+      write-paths:
+        - /
+
+parts:
+  busybox:
+    plugin: kbuild
+    source: https://github.com/asac/busybox.git
```

Comment:
Would it be a better idea to pull this repo into the ubuntu-core group so it doesn't belong to any one person (and be accidentally deleted)?